src/clusterfuzz/_internal/bot/webserver/http_server.py

```diff
@@ -146,5 +147,9 @@ def start():
   http_port_2 = environment.get_value('HTTP_PORT_2', 8080)
   if not port_is_open(http_host, http_port_1):
     start_server_thread(http_host, http_port_1)
+  else:
+    logs.log_warn(f"HTTP_PORT_1 ({http_port_1}) already open, not starting server thread.")
   if not port_is_open(http_host, http_port_2):
     start_server_thread(http_host, http_port_2)
+  else:
+    logs.log_warn(f"HTTP_PORT_1 ({http_port_1}) already open, not starting server thread.")
```

Comment:
s/1/2/ ?

Reply:
indeed sorry about that, I had those factorized initially.

Reply:
thanks!!!